README.md

```diff
@@ -293,3 +279,60 @@ func startOrchextraContentManager() {
 	}
 }
 ```
+
+### Third-party providers
+
+OCM provides a way to configure some Third-party providers by sending some information to the SDK. All providers can be configured in:
+
```

Comment:
as we're using provide and providers a bit too much here, i'd suggest to update to something like this:
OCM offers a way to configure third-party services and providers by setting their configuration data through the following property: